proto/dwtools/abase/l0/l8/gBuffer.s

```diff
@@ -2625,6 +2625,39 @@ function bufferResize_( dst, srcBuffer, size )
 
 
 //
+  /**
+   * The bufferBytesGet() routine converts and returns a new instance of U8x constructor( array of 8-bit unsigned integers ).
+   *
+   * @param { BufferRaw|BufferNode|BufferTyped|BufferView|String } src - Defined buffer or string.
+   *
+   * @example
+   * let src2 = new BufferRaw( 5 );
+   * _.bufferBytesGet(src2);
+   * // returns [ 0, 0, 0, 0, 0, ]
+   *
+   * @example
+   * let src1 = BufferNode.alloc( 5, 'a' );
+   * _.bufferBytesGet(src);
+   * // returns [ 97, 97, 97, 97, 97 ]
+   *
+   * @example
+   * let src3 = new I32x( [ 5, 6, 7 ] );
+   * _.bufferBytesGet(src3);
+   * // returns [ 5, 0, 6, 0, 7, 0 ]
+   *
+   * @example
+   * let src4 = 'string';
+   * _.bufferBytesGet(src4);
+   * // returns [ 115, 116, 114, 105, 110, 103 ]
+   *
+   *
+   * @returns { typedArray } Returns a new instance of U8x constructor.
+   * @function bufferBytesGet
+   * @throws { Error } Will throw an Error if { src } is not a typed array object or string.
```

Comment:
Please, remove begin of sentences `Will throw an Error`. 
The full sentence will look like 
```
@throws { Error } If {-src-} is not a typed array object or string.
```